package.json

```diff
@@ -9,6 +9,7 @@
     "build:packages": "lerna run build --no-private",
     "build:site": "lerna run build --scope=create-react-app",
     "postinstall": "lerna bootstrap --hoist --no-ci && npm run build:packages",
+    "tsd:hooks": "lerna exec tsd --scope=graphql-hooks",
```

Comment:
1. just name it tsd
2. use lerna run
3. create a tsd npm script in graphql-hooks

lerna will run that npm script in any package that has it, ignoring the others